### PR TITLE
Describe account endpoint

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

**Comment by ct-clearhaus** [comment]
RFR @soltveit

File: source/index.md [hunk 1]
```diff
@@ -628,6 +629,29 @@ authorization transaction and you supply card details.</p>
...
+<dl class="dl-horizontal">
+  <dt>merchant_id</dt>
+  <dd>Used for 3-D Secure and also for reference when talking to our support
+  staff. For 3-D Secure it is important to represent the number with the leading
+  zeros.</dd>
+  <dt>acquirer</dt>
+  <dd>Used for 3-D Secure.</dd>
+  <dt>text_on_statement</dt>
+  <dd>The default text_on_statement when nothing is sent for neither the
+  authorization nor the capture.</dd>
```
Comment (jsmpereira) [inline source/index.md]: double negative? :stuck_out_tongue:

Comment (jsmpereira) [inline source/index.md]: I stand corrected.

**Comment by soltveit** [comment]
:+1:

**Comment by ct-clearhaus** [comment]
@soltveit, can I ask you to deploy?

**Comment by soltveit** [comment]
@ct-clearhaus Sure

**Pull request merged**